packages/expo-image/android/src/main/java/expo/modules/image/Image.kt

```diff
@@ -0,0 +1,6 @@
+package expo.modules.image
+
+import android.graphics.drawable.Drawable
+import expo.modules.kotlin.sharedobjects.SharedRef
+
+class Image(ref: Drawable?) : SharedRef<Drawable?>(ref)
```

Comment:
```suggestion
class Image(ref: Drawable) : SharedRef<Drawable>(ref)
```
Can we make it not nullable?

Reply:
should be done